### PR TITLE
Speaking test module 

### PR DESCRIPTION
**Pull request by dimahalaiskiy**
fixed audiorecorder that user have only one record after each record

https://user-images.githubusercontent.com/74025448/127025132-e73d4d0a-1bd5-4845-8a16-02243701a4d5.mp4

### CODE REVIEW

**Comment by RomPavel** [comment]
Change target branch to dev

File: src/app/models/speaking-test/speaking-test.component.ts [hunk 1]
```diff
@@ -16,20 +16,23 @@ export class SpeakingTestComponent implements OnInit {
 
   chunks: Blob[] = [];
 
+  obj = {};
```
Comment (RomPavel) [inline src/app/models/speaking-test/speaking-test.component.ts:19]: remove this

**Comment by dimahalaiskiy** [comment]
https://user-images.githubusercontent.com/74025448/127170459-8af40e70-53f0-4726-9320-dad27afb04c7.mov

File: src/app/models/speaking-test/speaking-test.component.ts (hunk 1 above)
Comment (dimahalaiskiy) [reply src/app/models/speaking-test/speaking-test.component.ts]: ready yet

**Review by RomPavel — APPROVED** [review]

**Pull request merged**